src/SFA.DAS.LearnerDataMismatches.Web/Pages/_DataLockRow.cshtml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SFA.DAS.LearnerDataMismatches.Domain;
+using System;
+
+namespace SFA.DAS.LearnerDataMismatches.Web.Pages
+{
+    public class _DataLockRowModel
```

Comment:
Class name need not start with `_`

Reply:
Yeah, I was trying to match it to the `_DataLockRow` partial view.

But VS is complaining too.  I've removed the `_`.